Loop/Models/ServiceAuthentication/ShareService.swift

```diff
@@ -36,14 +38,23 @@ class ShareService: ServiceAuthentication {
                 value: url?.absoluteString,
                 options: [
                     (title: NSLocalizedString("US", comment: "U.S. share server option title"),
-                     value: DexcomShareURL.absoluteString)
+                     value: DexcomShareURL.absoluteString),
+                    (title: NSLocalizedString("Outside US", comment: "Outside US share server option title"),
+                     value: KnownShareServers.NON_US.rawValue)
+
                 ]
             )
         ]
 
+        if customServer.characters.count > 0 {
```

Comment:
I started doing that, but realized having an optional set to an initial state of nil would be more difficult to grasp for novice users not accustomed to editing code, instead of editing just the contents between two quotation marks. 

I guess i could use an optional with a more extensive description